pkgs/racket-doc/scribblings/reference/module-reflect.scrbl

```diff
@@ -535,6 +535,48 @@ If @racket[provided] is @|void-const|, then the module is
 @tech{visit}ed but not @tech{instantiate}d (see @secref["mod-parse"]),
 and the result is @|void-const|.}
 
+@bold{Note}: because dynamic-require is a procedure, it is necessary to wrap @racket[mod]
```

Comment:
Probably notes should be margin notes.

`dynamic-require` should be linked with `@racket[dynamic-require]`.